apptools/preferences/tests/test_py_config_file.py

```diff
@@ -49,7 +49,7 @@ def test_load_from_filename(self):
         self.assertEqual('blue', config['acme.ui']['bgcolor'])
         self.assertEqual(50, config['acme.ui']['width'])
         self.assertEqual(1.0, config['acme.ui']['ratio'])
-        self.assertEqual(True, config['acme.ui']['visible'])
+        self.assertTrue(config['acme.ui']['visible'])
```

Comment:
This might be worth keeping the original `assertEqual(True, ...)`.  This is loading data from the config file, and it is worth making sure this is actually a bool, not, e.g. `"true"`.